src/Glpi/Form/Destination/CommonITILField/RequestTypeField.php

```diff
@@ -0,0 +1,285 @@
+use Override;
+use Ticket;
+
+class RequestTypeField extends AbstractConfigField
+{
+    // Main configuration main
+    public const CONFIG_FROM_TEMPLATE = 'from_template_or_default';
+    public const CONFIG_SPECIFIC_VALUE = 'specific_value';
+    public const CONFIG_SPECIFIC_ANSWER = 'specific_answer';
+    public const CONFIG_LAST_VALID_ANSWER = 'last_valid_answer';
+
+    // Secondary config value that is used when the main value is CONFIG_SPECIFIC_VALUE
+    public const EXTRA_CONFIG_REQUEST_TYPE = 'specific_request_type';
+
+    // Secondary config value that is used when the main value is CONFIG_LAST_VALID_ANSWER
+    public const EXTRA_CONFIG_QUESTION_ID = 'specific_question_id';
+
+    #[Override]
+    public function getKey(): string
+    {
+        return 'request_type';
+    }
+
+    #[Override]
+    public function getLabel(): string
+    {
+        return __("Request type");
+    }
+
+    #[Override]
+    public function renderConfigForm(
+        Form $form,
+        mixed $configurated_value,
+        string $input_name,
+        array $display_options
+    ): string {
+        if (!is_array($configurated_value)) {
+            $configurated_value = $this->getDefaultValue($form);
+        }
+
+        $parameters = [
+            // Possible configuration constant that will be used to to hide/show additional fields
+            'CONFIG_SPECIFIC_VALUE'  => self::CONFIG_SPECIFIC_VALUE,
+            'CONFIG_SPECIFIC_ANSWER' => self::CONFIG_SPECIFIC_ANSWER,
+
+            // General display options
+            'options' => $display_options,
+
+            // Main config field
+            'main_config_field' => [
+                'label'           => $this->getLabel(),
+                'value'           => $configurated_value['value'] ?? $this->getDefaultValue($form)['value'],
+                'input_name'      => $input_name . "[value]",
+                'possible_values' => $this->getMainConfigurationValuesforDropdown(),
+            ],
+
+            // Specific additional confog for CONFIG_SPECIFIC_VALUE
+            'specific_value_extra_field' => [
+                'empty_label'     => __("Select a request type..."),
+                'value'           => $this->getConfiguratedSpecificValue($configurated_value),
+                'input_name'      => $input_name . "[" . self::EXTRA_CONFIG_REQUEST_TYPE . "]",
+                'possible_values' => Ticket::getTypes(),
+            ],
+
+            // Specific additional confog for CONFIG_SPECIFIC_ANSWER
+            'specific_answer_extra_field' => [
+                'empty_label'     => __("Select a question..."),
+                'value'           => $this->getConfiguratedQuestionId($configurated_value),
+                'input_name'      => $input_name . "[" . self::EXTRA_CONFIG_QUESTION_ID . "]",
+                'possible_values' => $this->getRequestTypeQuestionsValuesForDropdown($form),
+            ],
+        ];
+
+        $template = <<<TWIG
```

Comment:
Not sure where we draw the line regarding long inline twig.
Let me know if this would be better in a dedicated file.